source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -388,8 +371,11 @@ void BioXASSideTableCoordinator::manipulateVerticalPVChange(double upstreamHeigh
 
 void BioXASSideTableCoordinator::manipulateHorizontalPVChange(double upstreamOffset, double downstreamOffset, AMControl* lateralPV, AMControl* yawPV)
 {
-	double lateral = ( downstreamOffset + upstreamOffset ) / 2;
-	double yaw = radianToDegree(atan((downstreamOffset - upstreamOffset) / (tableHorizontalMotorPosition_ * 2)));
+	double lateral = calculateTableLateral(upstreamOffset, downstreamOffset);
+	double yaw = calculateTableYaw(upstreamOffset, downstreamOffset);
+
+	lateral = static_cast<double>(static_cast<int>(lateral*1000+0.5))/1000.0;
```

Comment:
That doesn't matter.  You could just pass the int and let the function do the implicit cast or use ```double(qRound(lateral))```.